### ISSUE TITLE
How to get 1.2.8 ?

### ISSUE THREAD

**Issue by Luuk34** [xavctn/img2table#164]
Trying to install img2table 1.2.8 I do:

```
D:\TEMP\python>pip show tesseract
WARNING: Package(s) not found: tesseract

D:\TEMP\python>pip show img2table
WARNING: Package(s) not found: img2table

D:\TEMP\python>pip install tesseract
Collecting tesseract
  Using cached tesseract-0.1.3-py3-none-any.whl
Installing collected packages: tesseract
Successfully installed tesseract-0.1.3

D:\TEMP\python>pip install img2table
Collecting img2table
  Using cached img2table-0.0.12-py3-none-any.whl (43 kB)
Collecting polars[pandas] (from img2table)
  Using cached polars-0.20.7-cp38-abi3-win_amd64.whl.metadata (15 kB)
Collecting pyarrow>=7 (from img2table)
  Using cached pyarrow-15.0.0-cp312-cp312-win_amd64.whl.metadata (3.1 kB)
.....
```

It will get version 0.0.12
```
D:\TEMP\python>pip list
Package         Version
--------------- --------
beautifulsoup4  4.12.3
bs4             0.0.2
img2table       0.0.12
numpy           1.26.4
opencv-python   4.9.0.80
pandas          2.2.0
pip             24.0
polars          0.20.7
pyarrow         15.0.0
PyMuPDF         1.23.21
PyMuPDFb        1.23.9
python-dateutil 2.8.2
pytz            2024.1
six             1.16.0
soupsieve       2.5
tesseract       0.1.3
tzdata          2023.4
XlsxWriter      3.1.9
```

How can I get the 1.2.8 version ?

I am using Windows 11, and did a new install of Python just before starting this.
```
D:\TEMP\python>python -V
Python 3.12.2
```

**Comment by xavctn** [comment]
Hello,
This is because the library does not support Python 3.12 as of now.
I am going to do some checks before allowing it.

**Comment by xavctn** [comment]
Hi,
The library should now be available on Python 3.12 with the latest releases
